services/apps/alcs/src/providers/typeorm/migrations/1693952313867-add_notification_statuses.ts

```diff
@@ -0,0 +1,37 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class addNotificationStatuses1693952313867
+  implements MigrationInterface
+{
+  name = 'addNotificationStatuses1693952313867';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE "alcs"."notification_submission_status_type" ("audit_deleted_date_at" TIMESTAMP WITH TIME ZONE, "audit_created_at" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(), "audit_updated_at" TIMESTAMP WITH TIME ZONE DEFAULT now(), "audit_created_by" character varying NOT NULL, "audit_updated_by" character varying, "label" character varying NOT NULL, "code" text NOT NULL, "description" text NOT NULL, "weight" smallint NOT NULL DEFAULT '0', "alcs_background_color" character varying NOT NULL, "alcs_color" character varying NOT NULL, "portal_background_color" character varying NOT NULL, "portal_color" character varying NOT NULL, CONSTRAINT "UQ_7fb8eeb106bb4bb0b0738bd1106" UNIQUE ("description"), CONSTRAINT "PK_9a28eacfd5a7b00f06bdf234bcd" PRIMARY KEY ("code"))`,
```

Comment:
looks like table comment is missing

Reply:
Will add comments when I get to documents.